ROADMAP.md

```diff
@@ -1,3 +1,29 @@
+
+## Roadmap Process
+
+wasmCloud roadmaps are planned quarterly (every three calendar months) in the first community meeting of the quarter. If there is a holiday conflict or a majority of maintainers and contributors will be absent from a community meeting, then the roadmap session can be postponed to the following Wednesday. The purpose of the planning session is to review the previous quarter's roadmap for successes and areas of improvement, as well as to plan initiatives for wasmCloud contributors in the following quarter. It is strongly recommended that a wasmCloud organization maintainer leads the roadmapping session as they are responsible for understanding all areas of the project.
+
+The process for wasmCloud quarterly planning is as follows, and can be done by any wasmCloud organization maintainer:
+1. Create a GitHub Discussion at https://github.com/wasmCloud/wasmCloud/discussions (e.g. [https://github.com/wasmCloud/wasmCloud/discussions/3813](https://github.com/wasmCloud/wasmCloud/discussions/3813)) for community comments, requests and discussion. This should be the primary feedback mechanism for bringing in work for the next quarter. Optionally, if there is a high level goal of a roadmap (e.g. revamp documentation, add support for a new language) communicate that goal in the GitHub discussion.
+1. Schedule the community meeting and set the primary agenda item to be `DISCUSSION: wasmCloud QX Roadmap` (e.g. [https://wasmcloud.com/community/2024/10/02/community-meeting](https://wasmcloud.com/community/2024/10/02/community-meeting))
+1. For each major repository in wasmCloud, create an issue milestone for the applicable quarter e.g. `wasmCloud Q424`. Name the milestones the same in each repository for assistance in roadmap creation.
+1. Before the community meeting, compile feedback from the GitHub discussion into GitHub issues to track individual action items. Issues should be placed on the `wasmCloud Roadmap` project and under the applicable issue milestone.
+1. In the [wasmCloud Roadmap GitHub project](https://github.com/orgs/wasmCloud/projects/7), create a `New View` that selects each issue from the quarter milestone, and name the roadmap `QX XXXX` e.g. `Q4 2024`.
+1. During the community meeting, review created issues from the discussion. At least 15 minutes of the community meeting should be left available for additional discussion for items that did not make it onto the GitHub discussion.
+1. After the community meeting, finalize the Roadmap by outlining the high level initiatives and the GitHub project on the [wasmCloud Roadmap](https://wasmcloud.com/docs/roadmap/) page, which serves as a place that onlookers can see the outcomes of the planning session.
```

Comment:
Add the "who?" here. Probably the org maintainer who started the process. This can probably be listed above the list by replacing the first line with something like

> The process for wasmCloud quarterly planning starts with one or more wasmCloud Org Maintainers volunteering or being assigned to be the planners. The maintainer should then follow the process below